replicatedhq/ship
# Clean up some linter errors, improve `ExitWithWarn` messaging

What I Did 
------------

Clean up some linter errors, improve `ExitWithWarn` messaging

How I Did it
------------

Don't print a message about an error on `ExitWithWarn`, since
the user declined a prompt.

How to verify it
------------

Run `ship init`, then run `ship init` again, and decline the "start from scratch" prompt. See that the following error is no longer printed:

```
There was an error configuring the application...
```

Description for the Changelog
------------

Clean up messaging when promting to use `ship update` instead of `ship init`

:canoe:


What I Did
------------


How I Did it
------------


How to verify it
------------


Description for the Changelog
------------



Picture of a Boat (not required but encouraged)
------------
